mesh/meshdb.go

```diff
@@ -714,10 +716,12 @@ func (m *MeshDB) Retrieve(key []byte, v interface{}) (interface{}, error) {
 	return v, nil
 }
 
-func (m *MeshDB) CacheWarmUp(from types.LayerID, to types.LayerID) error {
+func (m *MeshDB) cacheWarmUpFromTo(from types.LayerID, to types.LayerID) error {
 	m.Info("warming up cache with layers %v to %v", from, to)
 	for i := from; i < to; i++ {
-
+		if m.exit {
```

Comment:
this is not going to work, as exit is not protected and used concurrently.

Reply:
sure il make it atomic ot something

Reply:
thanks, supposed to be a chan.